### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ViktorKyrychenko**
Add Tests solutions from Task2

### CODE REVIEW

File: Assignment2/Assignment2/ViewController.swift [hunk 1]
```diff
@@ -9,36 +9,64 @@
...
-
+    
     //    Задача 4 Проверить, является ли заданное число совершенным
-    func isNumberPerfect(_ number: Int) {
-        fatalError()
+    func isNumberPerfect(number: Int) {
+        var deviderCount2 = 0
+        var deviderSum = 0
+        
+        for devider in 1..<number {
```
Comment (IvanBesarab) [inline Assignment2/Assignment2/ViewController.swift:57]: You can use number divided by two

File: Assignment3/Assignment3/ViewController.swift [hunk 2]
```diff
@@ -11,57 +11,161 @@ import UIKit
...
+                markNumbers += 1
+            }
+            if character.isLetter {
+                markLetter += 1
+            }
+            if character.isUppercase {
+                markUppercased += 1
+            }
+            if character.isPunctuation {
+                markSymbol += 2
```
Comment (IvanBesarab) [inline Assignment3/Assignment3/ViewController.swift:97]: White +2 not +1?

File: Assignment2/Assignment2/ViewController.swift (hunk 1 above)
Comment (ViktorKyrychenko) [reply Assignment2/Assignment2/ViewController.swift:57]: I thought to check all possible dividers there

File: Assignment3/Assignment3/ViewController.swift (hunk 2 above)
Comment (ViktorKyrychenko) [reply Assignment3/Assignment3/ViewController.swift:97]: As in the last password, there were only 4 possible variants to make +1, and in the task, you should get +5 in total, I understood it like +2 for Symbols

**Pull request closed without merging**